### ISSUE TITLE
upload the new Whole Word Masking pretrained models to Tensorflow Hub?

### ISSUE THREAD

**Issue by yliu2018** [google-research/bert#704]
When will the new whole world masking pretrained models be uploaded to Tensorflow Hub?

Thanks!
